kafka-ui-react-app/src/components/common/Input/InputIcon.styled.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { Colors } from 'theme/theme';
+
+interface Props {
+  className: string;
+  position: 'left' | 'right';
+  inputSize: 'M' | 'L';
+}
+
+const StyledIcon = styled.i<Props>`
+  position: absolute;
+  top: ${(props) => (props.inputSize === 'M' ? '8px' : '12px')};
```

Comment:
Let's use another approach
```
top: 50%;
transform: translate3d(0, -50%, 0);
```
Spacing with pixels is hardly maintainable and can break if the icon has another size.